rest/route/project_test.go

```diff
@@ -957,7 +979,7 @@ func (s *ProjectPutRotateSuite) TestRotateProjectVars() {
 	s.Contains(respMap["dimoxinil"], "banana")
 	s.Contains(respMap["dimoxinil"], "lemon")
 	s.Equal(resp.Status(), http.StatusOK)
-	s.Equal("yellow", s.sc.CachedVars[0].Vars["banana"])
+	//s.Equal("yellow", s.sc.CachedVars[0].Vars["banana"])
```

Comment:
Should this be commented out?

Reply:
CachedVars are an old property on the mock connector that isnt needed anymore, deleted now.